circe-checksum/build.gradle

```diff
@@ -34,8 +34,8 @@ task generateJniHeaders(type:Exec) {
     }
     dependsOn classes
     def classpath = sourceSets.main.output.classesDirs.join(":")
-    commandLine("javah", "-d", javahOutputDir, "-classpath",
-                classpath, "com.scurrilous.circe.crc.Sse42Crc32C")
+    commandLine("javac", "-h", javahOutputDir, "-classpath",
```

Comment:
fwiw, gradle has first class support for JNI headers now.  i didn't check the version, and i don't know gradle well enough yet to update the dependencies for the shared objects.

https://docs.gradle.org/current/dsl/org.gradle.api.tasks.compile.CompileOptions.html#org.gradle.api.tasks.compile.CompileOptions:headerOutputDirectory